onnxruntime/core/providers/webgpu/tensor/transpose.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/providers/webgpu/webgpu_supported_types.h"
+#include "core/providers/cpu/tensor/transpose.h"
+#include "core/providers/webgpu/webgpu_kernel.h"
+#include "core/providers/webgpu/program.h"
+
+namespace onnxruntime {
+namespace webgpu {
+
+class TransposeProgram final : public Program<TransposeProgram> {
+ public:
+  TransposeProgram(const std::string& kernel_name, const gsl::span<const size_t>& permutations)
```

Comment:
name can remove because it should always be "Transpose"

Reply:
done